run-all.sh

```
@@ -0,0 +1,148 @@
+
+echo -e "${GREEN}✓ 모든 프론트엔드 빌드 완료${NC}"
+
+# 1. MySQL 시작
+echo -e "\n${YELLOW}[1/6] MySQL 시작 중...${NC}"
+docker-compose up -d
+sleep 3
+echo -e "${GREEN}✓ MySQL 시작 완료${NC}"
+
+# 2. Eureka Server 시작
+echo -e "\n${YELLOW}[2/6] Eureka Server 시작 중...${NC}"
+cd eureka-server
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)